www.js

```diff
@@ -87,6 +87,7 @@ const server = http.createServer((req, res) => {
     return;
   }
   setSecurityHeaders(res);
+  res.setHeader('Content-Type', 'text/html; charset=utf-8');
```

Comment:
If we're serving other file types, e.g. `js` then this `Content-Type` header is incorrect?

Reply:
Could use [res.type](https://expressjs.com/en/api.html#res.type) to apply the appropriate `Content-Type`?